packages/twenty-front/src/modules/ui/navigation/navigation-drawer/components/NavigationDrawerSectionTitle.tsx

```diff
@@ -20,9 +21,12 @@ export const NavigationDrawerSectionTitle = ({
   label,
 }: NavigationDrawerSectionTitleProps) => {
   const loading = useIsPrefetchLoading();
```

Comment:
here I think we need to update useIsPrefetchLoading. I would actually set prefetchIsLoadedFamilyState atoms to true when we are in MockMode (this can be set in an effect)